thumbor_cloud_storage/loaders/cloud_storage_loader.py

```diff
@@ -1,16 +1,19 @@
 from tornado.concurrent import return_future
 from gcloud import storage
 from collections import defaultdict
+from oauth2client.service_account import ServiceAccountCredentials
 
 buckets = defaultdict(dict)
 
 @return_future
 def load(context, path, callback):
     bucket_id  = context.config.get("CLOUD_STORAGE_BUCKET_ID")
     project_id = context.config.get("CLOUD_STORAGE_PROJECT_ID")
+    auth_json = context.config.get("CLOUD_STORAGE_AUTH_JSON")
     bucket = buckets[project_id].get(bucket_id, None)
     if bucket is None:
-        client = storage.Client(project_id)
+        credentials = ServiceAccountCredentials.from_json_keyfile_name(auth_json)
```

Comment:
This should only instantiate and use credentials if auth_json is not None... @eceglov